core-bundle/src/Resources/contao/library/Contao/InsertTags.php

```diff
@@ -951,9 +951,9 @@ protected function doReplace($strBuffer, $blnCache)
 							$objFile = new File(rawurldecode($src));
 
 							// Add the image dimensions
-							if (($imgSize = $objFile->imageSize) !== false)
+							if (isset($objFile->imageSize[0], $objFile->imageSize[1]))
```

Comment:
It seems that `$objFile->imageSize` cannot be `false` anymore (it could be in the past). @ausi Is my replacement correct?

Reply:
Yes, is correct I think.